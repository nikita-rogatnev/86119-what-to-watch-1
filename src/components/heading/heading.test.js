import React from "react";
import Enzyme, {shallow} from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import toJson from "enzyme-to-json";

import Heading from "./heading";

Enzyme.configure({adapter: new Adapter()});

it(`Heading renders correctly`, () => {
  const tree = shallow(<Heading
    title={`Test title`}/>);

  expect(toJson(tree)).toMatchSnapshot();
});
