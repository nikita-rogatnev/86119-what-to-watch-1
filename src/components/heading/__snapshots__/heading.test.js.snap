// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading renders correctly 1`] = `
<div
  className="user-page"
>
  <withRouter(Connect(Header)) />
  <div
    className="error"
  >
    <h1>
      Test title
    </h1>
  </div>
  <Footer />
</div>
`;
